pyinterface/fastjet.i

```diff
@@ -139,9 +139,9 @@ static PyObject * FastJetError_;
 // include FastJetError in python module
 %pythoncode {
   if __package__ or "." in __name__:
```

Comment:
`__package__` has been removed from Python 3.14. `__spec__.parent` is the way to get this from Python 3.4+. Though doesn't this get generated by SWIG already? It feels like there a second layer of indirection here?

Reply:
is `__spec__.parent` supported back through python 3.9?